### PR TITLE
InputField DatePicker: Update min/max defaults

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#16986

*Description of changes:*
* Don't set date defaults when no min or max props dates are passed. (ie no range restriction)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/common/InputFieldDatePicker.vue [hunk 1]
```diff
@@ -86,10 +86,6 @@ export default defineComponent({
     initialValue: { type: String, default: '' },
     minDate: { type: String, default: '' },
     maxDate: { type: String, default: '' },
-    nudgeTop: { type: String, default: null },
-    nudgeBottom: { type: String, default: null },
-    nudgeRight: { type: String, default: null },
-    nudgeLeft: { type: String, default: null },
```
Comment (cameron-eyds) [inline ppr-ui/src/components/common/InputFieldDatePicker.vue:92]: The 'nudges' are no longer a part of vuetify, they use offset objects of which we are not really using anymore. 
Removing these until they are used/required.

**Review by dimak1 — APPROVED** [review]

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1650-evmfo4tu.web.app

**Review by dimak1 — APPROVED** [review]

**Review by doug-lovett — APPROVED** [review]
LGTM

**Review by RuoxuanPengBC — APPROVED** [review]
Looks great!

**Pull request merged**